build.gradle.kts

```diff
@@ -20,3 +21,10 @@ ktlint {
 }
 
 tasks.getByPath("ktlintCheck").shouldRunAfter("ktlintFormat")
+dependencies {
+    implementation(kotlin("stdlib-jdk8"))
+}
+
+kotlin {
+    jvmToolchain(8)
+}
```

Comment:
What does this line do?

Reply:
Again, it looks like a setting specific to JVM projects

Usually we indicate java version as

```
android {
    compileOptions {
        sourceCompatibility = JavaVersion.VERSION_1_8
        targetCompatibility = JavaVersion.VERSION_1_8
    }

    kotlinOptions {
        jvmTarget = "1.8"
    }
}
```